### PR TITLE
sg: Show progress bar when waiting for commands to install

### PR DESCRIPTION
**Pull request by mrnugget**
https://user-images.githubusercontent.com/1185253/150568922-69b2fa1d-1aaa-426b-a29d-cc86b49415d5.mp4

### CODE REVIEW

File: dev/sg/internal/run/run.go [hunk 1]
```diff
@@ -93,9 +93,9 @@ func Commands(ctx context.Context, globalEnv map[string]string, verbose bool, cm
 }
 
 func waitForInstallation(cmdNames map[string]struct{}, installed chan string, failures chan failedRun, okayToStart chan struct{}) error {
-	stdout.Out.Write("")
-	stdout.Out.WriteLine(output.Linef(output.EmojiFingerPointRight, output.StyleBold, "Waiting for %d commands to finish installation...", len(cmdNames)))
-	stdout.Out.Write("")
+	// stdout.Out.Write("")
+	// stdout.Out.WriteLine(output.Linef(output.EmojiFingerPointRight, output.StyleBold, "Waiting for %d commands to finish installation...", len(cmdNames)))
+	// stdout.Out.Write("")
```
Comment (danieldides) [inline dev/sg/internal/run/run.go:98]: Is there any reason to keep this code in and commented out?

**Review by bobheadxi — APPROVED** [review]
nice! also wondering about the commented line, but otherwise LGTM

File: dev/sg/internal/run/run.go (hunk 1 above)
Comment (bobheadxi) [reply dev/sg/internal/run/run.go:98]: Probably can be removed because there is a `Installing %d commands` message below!

Comment (mrnugget) [reply dev/sg/internal/run/run.go:98]: Yeah, sorry, I didn't clean up the code before opening the PR. Just opened it so it doesn't get lost :)

**Pull request merged**